--- IO/CGNS/vtkCGNSReader.cxx
+++ IO/CGNS/vtkCGNSReader.cxx
@@ -4222,17 +4222,6 @@
 }

 //----------------------------------------------------------------------------
-class WithinTolerance : public std::binary_function<double, double, bool>
-{
-public:
-  result_type operator()(first_argument_type a, second_argument_type b) const
-  {
-    bool result = (std::fabs(a - b) <= (a * 1E-6));
-    return (result_type)result;
-  }
-};
-
-//----------------------------------------------------------------------------
 int vtkCGNSReader::RequestData(vtkInformation* vtkNotUsed(request),
   vtkInformationVector** vtkNotUsed(inputVector), vtkInformationVector* outputVector)
 {